app/api/domains/cho.py

```diff
@@ -199,7 +211,10 @@ async def bancho_handler(
 
         return Response(
             content=login_data["response_body"],
-            headers={"cho-token": login_data["osu_token"]},
+            headers={
+                "cho-token": login_data["osu_token"],
+                "x-mcosu-features": f"submit={1 if app.settings.ALLOW_MCOSU_SCORES else 0}",
```

Comment:
Would more features be added in the future, and what's the serialization format for these if so?

Could we extract that into a function? (e.g. `def write_mcosu_features_header(allow_submission: bool) -> str: ...`)